subprojects/docs/src/samples/userguide/java/basic/groovy/build.gradle

```diff
@@ -58,8 +58,11 @@ dependencies {
 }
 
 javadoc {
-    options.docletpath = configurations.asciidoclet.files.toList()
+    dependsOn configurations.asciidoclet
     options.doclet = 'org.asciidoctor.Asciidoclet'
+    doFirst {
+        options.docletpath = configurations.asciidoclet.files.toList()
```

Comment:
This is bad, since the docletpath is then hidden for up-to-date checks. We should add a configure task instead.

Reply:
I thought we had up-to-date check for configurations, don't we?